dd-sdk-android-core/src/main/kotlin/com/datadog/android/core/DatadogCore.kt

```diff
@@ -487,8 +502,8 @@ internal class DatadogCore(
         internal const val EVENT_RECEIVER_ALREADY_EXISTS =
             "Feature \"%s\" already has event receiver registered, overwriting it."
 
-        const val LAST_VIEW_EVENT_DIR_MISSING_MESSAGE = "Directory structure %s for writing" +
-            " last view event doesn't exist."
+        const val NO_NEED_TO_WRITE_LAST_VIEW_EVENT = "No need to write last RUM view event: NDK" +
```

Comment:
minor: should this be public? It doesn't look like it's being accessed outside the internal scope at the moment